Tuxedo.js

```diff
@@ -62,6 +62,6 @@ Tux.Animations.Rotate.CounterClockRight = require('./Animations/Rotate/CounterCl
 Tux.Animations.Rotate.CounterClockUp = require('./Animations/Rotate/CounterClockUp');
 
 //warn user not to use this object in console
-console.warn('Using the Tux object directly is not recommended as it will mean loading all modules even those you don\'t intend to use.  Please review the TuxedoJS documents for requiring only the individual pieces of TuxedoJS that you will need.');
+console.warn('Using the Tuxx object directly is not recommended as it will mean loading all modules even those you don\'t intend to use.  Please review the TuxedoJS documents for requiring only the individual pieces of TuxedoJS that you will need.');
```

Comment:
We should discuss what we plan to do with the Tux vs. Tuxx naming conventions for variables.